alpharaw/wrappers/alphatims_wrapper.py

```diff
@@ -45,25 +45,26 @@ def import_raw(self, burker_d_folder: str):
 class AlphaTimsWrapper(TimsTOF):
     """Create a AlphaTims object that contains
     all data in-memory (or memory mapping).
-
-    Parameters
-    ----------
-    msdata : MSData_Base
-        The AlphaRaw data object.
-
-    dda : bool
-        If DDA, precursor indices will be equal to scan numbers.
-        If not DDA (i.e. DIA), precursor indices will be equal to the
-        scan number within a DIA cycle.
-
-    slice_as_dataframe : bool
-        If True, slicing returns a pd.DataFrame by default.
-        If False, slicing provides a np.int64[:] with raw indices.
-        This value can also be modified after creation.
-        Default is True.
     """
 
     def __init__(self, msdata: MSData_Base, dda: bool, slice_as_dataframe: bool = True):
+        """
+        Parameters
+        ----------
+        msdata : MSData_Base
+            The AlphaRaw data object.
+
+        dda : bool
+            If DDA, precursor indices will be equal to scan numbers.
+            If not DDA (i.e. DIA), precursor indices will be equal to the
+            scan number within a DIA cycle.
+
+        slice_as_dataframe : bool
```

Comment:
is this parameter even used somewhere?
(same for `_use_calibrated_mz_values_as_default`, `thermo_raw_file_name`, `bruker_d_folder_name`)

Reply:
See above.

Reply:
wont FIX

Reply:
See https://github.com/MannLabs/alphadia/blob/main/alphadia/data/alpharaw.py#L251

Reply:
> wont FIX:)

Lets MB keep it ..

Reply:
I see .. I would vote for removing it or moving the alphaDIA implmentation here in order to have less dead code

Reply:
Keep it in AlphaDIA as alphaDIA is better at handling numba jit classes or functions

Reply:
But I will still keep it in case alphaViz needs it

Reply:
`slice_as_dataframe` is required by the super class `AlphaTims`, it must be set here to enable alphatims functionalities.

Reply:
Done